.mvn/extensions.xml

```diff
@@ -3,11 +3,11 @@
 	<extension>
 		<groupId>org.eclipse.tycho.extras</groupId>
 		<artifactId>tycho-pomless</artifactId>
-		<version>1.3.0</version>
+		<version>1.5.0</version>
```

Comment:
For the sake of consistency, we should use version 1.5.1 because we use it in the Maven build.

Reply:
Done